plutus-core/pir/Main.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE RankNTypes #-}
+module Main where
```

Comment:
Can you put this in `plutus-core/executables`, which is where the others live now?

Reply:
I think this is fine as it is for now, but we probably want to bring it more in line with the other executables when we have time, cc @thealmarty